Doc/library/pathlib.rst

```diff
@@ -552,7 +552,7 @@ Pure paths provide the following methods and properties:
       Traceback (most recent call last):
         File "<stdin>", line 1, in <module>
         File "pathlib.py", line 694, in relative_to
```

Comment:
The line # is obsolete.  Could change.  In 3.10/11
```suggestion
        File "pathlib.py", line 818, in relative_to
```
was 941 in 3.9 but not worth the bother to change in backport.